h/templates/client/viewer.html

```diff
@@ -40,5 +40,23 @@
       ng-repeat="child in threadRoot.children | orderBy : sort.predicate"
       ng-show="vm.shouldShow()">
   </li>
+  <li class="loggedout-message" ng-if="isSidebar && auth.status === 'signed-out' && settings.annotations && selectedAnnotationAvailable()" ng-cloak>
```

Comment:
I would suggest extracting the `ng-if` condition out of the template and into a separate function in the controller, where you can add comments to explain what is going on and tests that the result is correct. Otherwise we're going to forget what these conditions are for in future.